package java.renameMethod.v0;

public class User {
  private int age;

  public User(int age) {
    this.age = age;
  }

  public boolean isOkay() {
    final int THRESHOLD = 18;
    return age >= THRESHOLD;
  }
}
